packages/date-picker/src/DatePicker.tsx

```diff
@@ -19,38 +19,34 @@ export const DatePickerWrapper: React.FunctionComponent<DatePickerProps> = ({
   onDayChange,
   labelText,
   isDisabled = false,
-}) => (
-  <>
-    {labelText && <Label disabled={isDisabled} labelText={labelText} />}
-    <div
-      className={classnames(
-        styles.wrapper,
-        styles.withStartIconAdornment,
-        styles.withSearch,
-        {
-          [styles.withDisabled]: isDisabled,
+}) => {
+  const getNavbar = ({ ...navbarProps }: CalendarNavProps) => (
+    <CalendarNav {...navbarProps} />
+  )
+
+  return (
+    <>
+      <TextField
```

Comment:
You'll need to be pretty careful about how people will use this component if you don't expose all these `TextField` props. Folks generally like the ability to do things like add their own automation/testing id, refs, and so on. Take a look at the [props on TextField](https://github.com/cultureamp/kaizen-design-system/blob/master/draft-packages/form/KaizenDraft/Form/TextField/TextField.tsx#L35-L50) and think about the use case for each prop and whether a consumer would like similar functionality from this component too.

Reply:
Along the same lines, should we ever have a hardcoded `id`? See `id="date-picker"` below. I can see a datepicker used multiple times on one page and then we'd have the same ID multiple times which could potentially lead to weird, unexpected behavior.

Reply:
Thanks for that. I've passed through `id` along with the other props I think a consumer would want.